src/actions/animalActions.spec.js

```diff
@@ -0,0 +1,196 @@
+  const mockStore = configureMockStore(middleware);
+
+  describe('Enviar el formulario de un animal con acciones asíncronas', () => {
+    afterEach(() => {
+      nock.cleanAll();
+    });
+
+    describe('Exito al enviar el formulario del animal', () => {
+      const animal = {
+        chip_num: '1',
+        species_id: '1',
+        sex: 'female',
+        admission_date: '2010-10-10',
+        name: 'Test',
+        birthdate: '2010-10',
+        race: 'beagle',
+        death_date: '',
+        castrated: false,
+        vaccines: false,
+        profile_image: 'image_data',
+        weight: 15
+      };
+
+      it('Devuelve la acción sendAnimalFormSuccess con sendAnimalForm', () => {
+        nock(consts.API_STAGING_URL)
+          .post('/animals', parseAnimal(animal))
+          .reply(201, { id: 43 });
+
+        const expectedAction = [{
+          type: 'SEND_ANIMAL_FORM_SUCCESS', response: { id: 43 }
+        }];
+
+        const store = mockStore(initialState.animalForm);
+
+        return store.dispatch(animalActions.sendAnimalForm(animal))
+          .then(() => {
+            expect(store.getActions()).to.deep.equal(expectedAction);
+          });
+      });
+
+      it('La acción sendAnimalFormSuccess cambia a success en el store', () => {
+        nock(consts.API_STAGING_URL)
+          .post('/animals', parseAnimal(animal))
+          .reply(201, { id: 43 });
+
+        const expectedAction = {
+          type: 'SEND_ANIMAL_FORM_SUCCESS', response: { id: 43 }
+        };
+
+        const store = createStore(rootReducer, initialState);
+
+        store.dispatch(expectedAction);
+        expect(store.getState().animalForm.success).to.equal(true);
+      });
+    });
+
+    describe('Error al enviar el formulario del animal', () => {
+      const animal = {
+        chip_num: '1',
+        species_id: '1',
+        sex: 'female',
+        admission_date: '2010-10-10',
+        name: '',
+        birthdate: '2010-10',
+        race: 'beagle',
+        death_date: '',
+        castrated: false,
+        vaccines: false,
+        profile_image: 'image_data',
+        weight: 15
+      };
+
+      it('Devuelve la acción sendAnimalFormError con sendAnimalForm cuando el nombre esta vacío', () => {
+        nock(consts.API_STAGING_URL)
+          .post('/animals', parseAnimal(animal))
+          .reply(422, { error: { name: 'no puede estar en blanco' } });
+
+        const store = mockStore(initialState.animalForm);
+
+        const errors = 'no puede estar en blanco';
+        const expectedAction = [{
+          type: types.SEND_ANIMAL_FORM_ERROR, errors: { error: { name: errors } }
+        }];
+
+        return store.dispatch(animalActions.sendAnimalForm(animal))
+          .then(() => {
+            expect(store.getActions()).to.deep.equal(expectedAction);
+          });
+      });
+
+      it('La acción sendAnimalFormError guarda en el store el error', () => {
+        nock(consts.API_STAGING_URL)
```

Comment:
este `nock` lo usas en este `it` y en el de arriba, fijate si podes correrlo para el `it` padre, luego de definir el animal